web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc

```diff
@@ -0,0 +1,137 @@
+= Web Service Consumer Connector Examples - Mule 4
+
+The following example shows how to transform a variable value into XML and pass that value to the Web Service Consumer Connector that subsequently consumes a web service.
+
+.Web Service Consumer Connector Example Flow
+image::web-service-consumer-example1.png[Web Service Consumer Example Flow]
+
+== Web Service Consumer Example in Anypoint Studio
+To create the flow in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Core* > *Scheduler*.
+. Drag *Scheduler* component to the Studio canvas. +
+The *Scheduler* component initiates a flow when a time-based condition is met.
+. Set the *Display Name* field to `Scheduler`.
+. Set the *Scheduling Strategy* to `Fixed Frequency`, *Frequency* to `1000`, *Start delay* to `0` and *Time unit* to `MILLISECONDS (Default)`.
+
+In Studio, the *Scheduler* configuration looks like this:
+
+.Scheduler Configuration
+image::web-service-consumer-example2.png[Scheduler Configuration]
+
+[start=5]
+. Drag a *Set Variable* component onto the canvas, to the right of the *Scheduler* component. +
+The *Set Variable* component creates the variable to be transformed into XML later in the flow.
+. Set the *Display Name* field to `Set Variable`.
+. Set *Name* to `variableSample` and *Value* to `test`.
+
+In Studio, the *Set Variable* configuration looks like this:
+
+.Set Variable Configuration
+image::web-service-consumer-example3.png[Set Variable Configuration]
+
+[start=8]
+. Drag a *Transform Message* component onto the canvas, to the right of the *Set Variable* component. +
+The *Transform Message* component transforms the variable value into XML.
+. In the *Output* field of the *Transform Message* component configuration, set the DataWeave expression that:
+* Transforms the variable value output to `application/xml`.
+* Defines the name of the web service operation that the Web Service Consumer aims to invoke `attachOperation`.
```

Comment:
I got a bit confused by capping web service consumer (did you mean the connector name?) and the use of "aims to", so...:
```suggestion
* Defines the name of the web service operation invoked by `attachOperation`.
```
?